samples/portable-sdks/python/sub-orchestrations-with-fan-out-fan-in/requirements.txt

```diff
@@ -0,0 +1,3 @@
+durabletask
```

Comment:
technically not necessary because it is a direct requirement of `durabletask-azuremanaged`